lib/Drupal/Core/StringTranslation/TranslationInterface.php

```diff
@@ -70,19 +70,66 @@ public function translate($string, array $args = array(), array $options = array
-   *   themed. See format_string(). Note that you do not need to include @count
-   *   in this array; this replacement is done automatically for the plural case.
+   *   themed. See \Drupal\Component\Utility\SafeMarkup::format(). Note that you do
+   *   not need to include @count in this array; this replacement is done
+   *   automatically for the plural cases.
    * @param array $options
    *   An associative array of additional options. See t() for allowed keys.
    *
    * @return string
    *   A translated string.
    *
-   * @see self::translate
-   * @see \Drupal\Component\Utility\String
+   * @see self::translate()
    * @see t()
-   * @see format_string()
+   * @see \Drupal\Component\Utility\SafeMarkup::format()
+   * @see self::formatPluralTranslated
    */
   public function formatPlural($count, $singular, $plural, array $args = array(), array $options = array());
 
+  /**
+   * Formats an already translated string containing a count of items.
+   *
+   * This function ensures that the string is pluralized correctly. As opposed
+   * to the formatPlural() method, this method is designed to be invoked with
+   * a string already translated (such as with configuration translation).
+   *
+   * @param int $count
+   *   The item count to display.
+   * @param string $translation
+   *   The string containing the translation of a singular/plural pair. It may
+   *   contain any number of possible variants (depending on the language
+   *   translated to) separated by the value of the LOCALE_PLURAL_DELIMITER
+   *   constant.
+   * @param array $args
+   *   Associative array of replacements to make in the translation. Instances
+   *   of any key in this array are replaced with the corresponding value.
+   *   Based on the first character of the key, the value is escaped and/or
+   *   themed. See \Drupal\Component\Utility\SafeMarkup::format(). Note that you do
+   *   not need to include @count in this array; this replacement is done
+   *   automatically for the plural cases.
+   * @param array $options
+   *   An associative array of additional options. The 'context' key is not
+   *   supported because the passed string is already translated. Use the
+   *   'langcode' key to ensure the proper plural logic is used.
+   *
+   * @return string
+   *   The correct substring for the given $count with $args replaced.
+   *
+   * @see self::formatPlural()
+   * @see \Drupal\Component\Utility\SafeMarkup::format()
+   */
+  public function formatPluralTranslated($count, $translation, array $args = array(), array $options = array());
+
+   /**
+    * Returns the number of plurals supported by a given language.
+    *
+    * @param null|string $langcode
+    *   (optional) The language code. If not provided, the current language
+    *   will be used.
+    *
+    * @return int
+    *   Number of plural variants supported by the given language.
+    */
+   public function getNumberOfPlurals($langcode = NULL);
```

Comment:
https://github.com/LionsAd/service_container/blob/7.x-1.x/src/StringTranslation/StringTranslation.php

Will need updating for the new methods. Thanks Scrutinizer!